data/libraries.json

```diff
@@ -757,5 +757,14 @@
             "JavaScript"
         ],
         "url": "http://ivan.vucica.net"
+    },
+    {
+        "doap": "https://git.sr.ht/~nicoco/slidge/tree/master/item/doap.xml",
+        "last_renewed": "2023-01-18T00:00:00",
+        "name": "Slidge",
```

Comment:
Please make sure to keep it either `Slidge` or `slidge`. This differs between this entry and Slidge's DOAP file.